news/2022-04-21-osu-catch-world-cup-2022-registrations-now-open.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "osu!catch World Cup 2022: Registrations now open!"
+date: 2022-04-21 10:00:00 +0000
+---
+
+Summoning all fruit catchers:the osu!catch World Cup is back for its 11th instalment!
+
+![](/wiki/Tournaments/CWC/2022/img/cwc2022_banner@2x.jpg)
+
+The osu!catch World Cup brings with it a slew of changes, from the tournament format to the pool structure, no stone has been left unturned. Teams will play a Qualifier stage, where 9 maps will decide the who dashes to the top seeds for a Round of 32, double elimination bracket. Furthermore, all pools after Qualifiers will feature the ubiquitous Mixed Mod bracket.
+
+For more details on these changes, please refer to the **[osu!catch World Cup 2022 wiki article](/wiki/Tournaments/CWC/2022)**. You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1562118)** or at the public **[community server](https://discord.gg/0Vxo9AsejDkGlk3H)**.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | Apr 21st – May 5th |
+| Qualifier Showcase | May 14th (14:00 UTC) |
+| Qualifier Stage | May 21st & May 22nd |
+| Round of 32 | May 28th & May 29th |
+| Round of 16 | Jun 4th & Jun 5th |
+| Quarterfinals | Jun 11th & Jun 12th |
+| Semifinals | Jun 18th & Jun 19th |
+| Finals | Jun 25th & Jun 26th |
+| Grand Finals | Jul 2nd & Jul 3rd |
+
+## Prizes
+
+The osu!catch World Cup 2022 offers a $2,000 cash prize pool and limited edition merch!
+
+- **1st:** 48% of the prize pool, single-run merch, profile badge, **osu!catch Champion** user title for one year
+- **2nd:** 32% of the prize pool, single-run merch, profile badge
+- **3rd:** 20% of the prize pool, single-run merch, profile badge
+
+## How do I register?
+
+- **[Click here to sign up!](https://osu.ppy.sh/community/tournaments/33)**
+- Do not create teams in advance! Teams will be formed by each country's team captain after the Registration Phase is concluded.
+- Submit your registration before the **5th of May, 23:59 UTC** — late registrations will not be allowed under any circumstances!
+
+Please note that you may only participate if your **osu!catch Global Ranking is #5000 or higher, and you have not violated the [osu! community rules](/wiki/Rules) in the past 12 months**.
+
+If your registration passes our internal checks, you will be placed on the candidate list for your country. When that country's captain has been assigned, the list will be sent to them. This may take a while, so please be patient.
```

Comment:
```suggestion
If your registration passes our internal checks, you will be placed on the candidate list for your country. When the country's captain has been assigned, the list will be sent to them. This may take a while, so please be patient.
```